File: classes/map/Map.py
from classes.map.Tile import Tile

class Map:
	def __init__(self, playerCount):
		initialBoardSize = 3
		self.board = [[Tile() for i in range(initialBoardSize+playerCount)] for j in range(initialBoardSize+playerCount)]
		
	def getTile(self, x, y):
		return self.board[x][y]

File: Assets/Classes/Map/Map.cs
public class Map {
	public Tile[,] board;

	public Map(int playerCount) {
		int initialBoardSize = 3;
		initializeBoard (initialBoardSize * playerCount);
	}

	private void initializeBoard(int size) {
		board = new Tile[size, size];
		for (int i = 0; i < size; i++) {
			for (int j = 0; j < size; j++) {
				board [i,j] = new Tile ();
			}
		}
	}
}
